Controllers/HomeController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Zoltu.Bags.Client.Controllers
+{
+	public class HomeController : Controller
+	{
+		[Route("", Name ="Home", Order = 0)]
+		[Route("app", Name = "app", Order = 0)]
+		public IActionResult App()
+		{
+			ViewData["url"] = "https://bagcupid.com/";
+			ViewData["type"] = "website";
+			ViewData["title"] = "Bag Cupid";
+			ViewData["description"] = "What is your dream bag? Are you having trouble finding it? Let us help you!";
+
+			return View("Index");

```

Comment:
This needs to be lower case index because the server runs on Linux which is a case-sensitive operating system, so index.html and Index.html are not the same thing, which results in an exception being thrown here.